superset-frontend/src/features/alerts/buildErrorTooltipMessage.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { styled } from '@superset-ui/core';
+import { ValidationObject } from './types';
+
+const StyledList = styled.ul`
+  margin: 0;
+  padding: 0;
+  li {
+    padding-left: 12px;
+  }
+`;
+
+export const buildErrorTooltipMessage = (
+  build = true,
+  setErrorTooltipMessage: Function,
+  validationStatus: ValidationObject,
+) => {
+  if (build) {
+    const sectionErrors: string[] = [];
+    Object.values(validationStatus).forEach(validationData => {
+      if (!validationData.status) {
+        const sectionTitle = `${validationData.name}: `;
+        sectionErrors.push(sectionTitle + validationData.errors.join(', '));
+      }
+    });
+    setErrorTooltipMessage(
+      <div>
+        Not all required fields are complete. Please provide the following:
+        <StyledList>
```

Comment:
Dots in the list are too far left

<img width="286" alt="image" src="https://github.com/apache/superset/assets/15073128/cadc7fe1-1d97-422c-8ced-854acc657ebd">